docs/lib/auth/fragments/flutter/signin_web_ui/20_platform_specific_setup.md

```diff
@@ -0,0 +1,45 @@
+## How It Works
+Signin with web UI requires the Amplify plugin to display the signin UI inside a webview. This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app.
+
+## Android Platform Setup
+Add the following activity to the `AndroidManifest.xml` file in your app's `Android` directory, replacing `myapp` with whatever value you used for your redirect URI prefix:
+
+  ```xml
+  <activity
+      android:name="com.amazonaws.mobileconnectors.cognitoauth.activities.CustomTabsRedirectActivity">
+      <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+
+          <data android:scheme="myapp" />
+      </intent-filter>
+  </activity>
+  ```
+
+## iOS Platform Setup
+Add the following entry to the URL scheme in the `Info.plist` file in your app's `iOS` directory, replacing `myapp` with whatever value you used for your redirect URI prefix:
```

Comment:
Same question -- is the directory really stylized with some upper case letters, or is it `ios`?
```suggestion
Add the following entry to the URL scheme in the `Info.plist` file in your app's `iOS` directory. Replace `myapp` with the "redirect signin URI" you provided to the CLI:
```

Reply:
Pushing similar revision from local